agoric/src/httpSender.ts

```diff
@@ -0,0 +1,16 @@
+import axios from 'axios'
+
+export interface Action {
+  type: string
+  data: unknown
+}
+export interface HTTPSenderReply {
+  status: number
+  response: unknown
+}
+export type HTTPSender = (obj: Action) => Promise<HTTPSenderReply>
+
+export const makeHTTPSender: (url: string) => HTTPSender = (url) => async (obj) => {
+  const response = await axios.post(url, obj)
```

Comment:
You can make use of the `Requester.request` function instead of using axios directly. It just a layer on top of `axios` with some helpers.

Reply:
Done!